File: dataset/output/MISTRAL_codestral-latest/Task74_MISTRAL_codestral-latest.java
// Java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class PostgresConnection {
    public static Connection getConnection() throws SQLException {
        String url = "jdbc:postgresql://localhost/test";
        String user = "postgres";
        String password = "password";
        return DriverManager.getConnection(url, user, password);
    }
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task74_MISTRAL_codestral-latest.cpp
// C++
#include <libpq-fe.h>
#include <iostream>

PGconn* getConnection() {
    const char *conninfo = "host=localhost dbname=test user=postgres password=password";
    PGconn *conn = PQconnectdb(conninfo);

    if (PQstatus(conn) != CONNECTION_OK) {
        std::cerr << "Connection to database failed: " << PQerrorMessage(conn) << std::endl;
        PQfinish(conn);
        return NULL;
    }

    return conn;
}